tests/serializers/test_definitions_recursive.py

```diff
@@ -76,3 +76,20 @@ def test_custom_ser():
         'name': 'root',
         'sub_branch': "{'name': 'branch', 'sub_branch': None}",
     }
+
+
+def test_recursive_function():
```

Comment:
Just to be clear, without the change to `function.rs` from this PR, the following attempt to initialize the SchemaSerializer produces a `Duplicate schema ref` error.

Reply:
This test looks good to me 👍🏻